iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/SecureParameterDefaultValueCheck.java

```diff
@@ -74,20 +73,10 @@ private static Predicate<Expression> isSensitiveDefaultValue() {
   }
 
   private static Predicate<Expression> isSecureParameterReference() {
-    return expr -> {
-      if (expr.is(ArmTree.Kind.IDENTIFIER)) {
-        // ARM Bicep
-        Identifier identifier = (Identifier) expr;
-        return isSecureParameter(identifier, identifier.value());
-      } else if (expr.is(ArmTree.Kind.FUNCTION_CALL)) {
-        // ARM Json -- TODO SONARIAC-1405: ARM template expressions: replace `variables()` and `parameters()` with corresponding Identifiers
-        var parameterName = CheckUtils.parameterName((FunctionCall) expr);
-        if (parameterName != null) {
-          return isSecureParameter(parameterName, parameterName.value());
-        }
-      }
-      return false;
-    };
+    // FixMe: should be checked for Parameter, but in Bicep parameters are Variables for now
+    return expr -> expr instanceof HasIdentifier hasIdentifier &&
+      hasIdentifier.identifier() instanceof Identifier identifier &&
+      isSecureParameter(identifier, identifier.value());
```

Comment:
How do you handle the case `parameter(unknown_expression)`? I think we will raise an FP at the moment. If yes, I think we should create a ticket to handle this.

Reply:
Do you mean the case when `unknown_epxression` evaluates to a secure parameter name? Yes, that would be a FP. I'll create a ticket